javascript/javascript2/week1/index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Spice Trails</title>
+  </head>
+  <body>
+    <header class="header">
+      <img
+        class="logo"
+        src="https://raw.githubusercontent.com/Elnaz-Talebi/team32-homework/931068cc134d5b11cdf2b538130a769d16cd40db/javascript/javascript2/week1/Pictures/logo%20web.png"
+      />
+      <h1>Spice Trails</h1>
+      <nav>
+        <ul>
+          <li><a href="#about">About</a></li>
+          <li><a href="#recipe">Recipe</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+    <section id="about" class="top-page">
+      <p class="about-me">
+        Welcome to Spice Trails, the ultimate destination for food lovers! Here,
+        anyone can share their favorite recipes and discover new dishes from
+        around the world. Explore recipes by category, connect with fellow food
+        enthusiasts, and bring delicious creations to life in your kitchen.
+        Whether you're a seasoned chef or a home cook, there's a place for you
+        here!
+      </p>
+      <input class="search-box" type="text" placeholder="Search.." />
+    </section>
+    <section>
+      <form id="recipe-form">
+        <fieldset class="form_section">
+          <label for="title"
+            >Title:<input name="title" placeholder="Title of your recipe"
+          /></label>
```

Comment:
Watch out for the input field. It doesn't have an` id="title"` even though the `<label>` references `"title"`. This breaks the label-input asocciation and affects accessibility and form usability.